tests/end2end/features/keyinput.feature

```diff
@@ -162,3 +162,44 @@ Feature: Keyboard input
         And I press the key "c"
         Then the message "foo 3" should be shown
         And the message "foo 3" should be shown
+
+    # :test all tabs.mode_on_change modes
```

Comment:
I don't understand this comment. Why `:test`?

Reply:
no idea... probably a typo. :)